homework-submission/homework-week1/src/App.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import Todos from './todos';
+import todosData from './data/todos.json'
+import './App.css';
+
+
+class App extends Component {
+  state = {
+    todos: todosData
+  }
+
+  deleteTodo = (id) => {
+    const todos = this.state.todos.filter(todo => {
+      return todo.id !== id
+    });
+    this.setState({
+      todos
+    })
+  }
+
+  checkBoxHandler = (id) => {
+    const todos = this.state.todos.map(item => {
+      if (item.id === id) {
+        return {
+          ...item, done: !item.done
```

Comment:
+1 for having two clean functions and they work great(deleteTodo and checkboxHandler)